packages/core/utils/lib/webhook.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const webhookEvents = {
+  ENTRY_CREATE: 'entry.create',
+  ENTRY_UPDATE: 'entry.update',
+  ENTRY_DELETE: 'entry.delete',
+  ENTRY_PUBLISH: 'entry.publish',
+  ENTRY_UNPUBLISH: 'entry.unpublish',
+  MEDIA_CREATE: 'media.create',
+  MEDIA_UPDATE: 'media.update',
+  MEDIA_DELETE: 'media.delete',
+};
+
+/**
+ * TODO V5: remove this file
+ * @deprecated
```

Comment:
```suggestion
 * TODO V5: remove this file
 *
 * @deprecated Use `strapi.webhookStore.listAllowedEvents()` instead.
```

Also I think it would have been nice to add webhookStore definition + new methods in the API ref doc @jhoward1994  wdyt?

Reply:
+1

Reply:
yes, that looks like a good idea!

Reply:
Any ETA or workaround on this ?

Reply:
We plan to release this next monday, and I would say the workaround would be to use `strapi.webhookStore.listAllowedEvents()` :)